src/ng-multiselect-dropdown/src/multi-select.component.html

```diff
@@ -25,7 +25,7 @@
     <ul class="item2" [style.maxHeight]="_settings.maxHeight+'px'">
       <li *ngFor="let item of _data | multiSelectFilter:filter; let i = index;" (click)="onItemClick($event,item)" class="multiselect-item-checkbox">
         <input type="checkbox" aria-label="multiselect-item" [checked]="isSelected(item)" [disabled]="disabled || (isLimitSelectionReached() && !isSelected(item)) || item.isDisabled" />
-        <div>{{item.text}}</div>
+        <div><i class="{{item.icon}}"></i> {{item.text}}</div>
```

Comment:
Anything related to icons would need to be tested first.  You have modified the code as if 100% of users would consume this property.  It would be an optional property.  
<i  *ngIf="item?.icon" class="{{item.icon}}"></i>